### PR TITLE
Add Campus Safety Category to CSV Download (fixes #108)

### LINKED ISSUE #108: Add Campus Safety Category to Download CSV Output File
### Expected behavior
All table columns plus location (lat/long) will be output when downloading a CSV using the blue "Download CSV" button.

### Actual behavior
The following columns are exported:
address	category	date	descript	incidntnum	location	pddistrict	resolution	time

### Steps to reproduce
Run a search that returns results.
Download CSV.
Review columns exported.

### Steps to fix
Add Campus Safety Category (aka CSCategory) to the Download CSV output.

### PR DESCRIPTION
**Pull request by squanto**
https://github.com/sfbrigade/sf-crime-data/issues/108

Try it out at: http://hugomelo.com/sf-crime-data/

### CODE REVIEW

**Comment by paulhennings** [comment]
Hugo, Please send a link to your personal hosting where we can test this change (second set of eyes).

File: js/modules/csv.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            if (rowIndex < rows.length) {
+                link += "\n";
+            }
+            return link;
+        }, csvLink)
+    }
+
+    function _downloadCsv(dataLink) {
+        var evt = document.createEvent("MouseEvents");
+        evt.initMouseEvent("click", true, true, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
```
Comment (JoeGaebel) [inline js/modules/csv.js]: Not sure what this is all about. Seems like we're tracking the click to handle the button click. If that's the case, I think there's better was of binding a button click to a js function.

**Comment by paulhennings** [comment]
There seems to be a sequence or state problem here. You have to click the Download CSV button two times to get the current information. Recreate as follows:
1. Launch http://hugomelo.com/sf-crime-data/ (default search for 267 Buena Vista Ave E displays).
2. Click Download CSV. Nothing happens.
3. Click Download CSV a second time. You get the correct data download. (Note number of rows returned. For 08/04/2017 - 09/03/2017, 42 rows are returned for the default search both on screen and in CSV download).
4. Do a new search by drawing a polygon on the map.
5. Click Download CSV. The download contains the search data from the prior search (default search for 267 Buena Vista Ave E).
6. Click Download CSV a second time. The current (polygon) search data downloads.
Same thing happens with a radial search, so I don't think it's unique to the polygon/rectangle logic path v. the radial logic.
-------------------------
Closing without pulling, to clear the path for evaluation of other pull requests.
Please submit a pull request again after testing good.
Be sure to merge in any other changes to the production gh-pages as we go forward.
Thanks!

File: js/modules/csv.js (hunk 1 above)
Comment (squanto) [reply js/modules/csv.js]: Hey @JoeGaebel , sorry for the delayed response.

This lil bit creates a new anchor tag and a click event. It then triggers the click event on the link. For sure there are better ways of binding a button click to a function. The goal of this bit is just to name the file, set it's contents, and download it. 

Edit: these browser api's are actually super deprecated as well

**Pull request closed without merging**